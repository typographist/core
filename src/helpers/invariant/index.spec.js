import { invariant } from '.';

describe('helpers', () => {
  describe('invariant fn', () => {
    it('if the condition is not fulfilled, throw an error', () => {
      expect(invariant(() => false, 'WTF!?')).toEqual(undefined);
    });
  });
});
