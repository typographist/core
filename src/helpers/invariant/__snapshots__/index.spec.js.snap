// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`helpers invariant fn if the condition is not fulfilled, throw an error 1`] = `undefined`;
